lib/internal/Magento/Framework/Mail/Template/TransportBuilder.php

```diff
@@ -172,12 +172,27 @@ public function setReplyTo($email, $name = null)
 
     /**
      * Set mail from address
+     * 
+     * @deprecated Use setFromByStore
```

Comment:
1. It's better as a comment for `@deprecated` define reason why it was deprecated and use `@see` tag for reference with new method.
2. It's better to use new method with "null" value for $store inside deprecated method to reduce code duplication

Reply:
Done